interface/controller/app_controller.go

```diff
@@ -0,0 +1,54 @@
+package controller
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strconv"
+
+	"github.com/lscantillowl/academy-go-q42021/domain/model"
+)
+
+func respondWithJSON(w http.ResponseWriter, code int, payload interface{}) {
+	response, _ := json.Marshal(payload)
+
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(code)
+	w.Write(response)
+}
+
+func HomeHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
By convention, every exported stuff (function, variable, constant, etc) should have a description

```go
// HomeHandler - Description here....
func HomeHandler(w http.ResponseWriter, r *http.Request) {
```